src/main/java/org/apache/fineract/operations/TransactionRequestSpecs.java

```diff
@@ -45,10 +44,20 @@ public static <T> Specifications<TransactionRequest> in(SingularAttribute<Transa
         }));
     }
 
-    /*public static Specifications<TransactionRequest> filterByErrorDescription(List<String> errorDescriptions) {
+    public static Specifications<TransactionRequest> filterByErrorDescription(List<String> errorDescriptions) {
```

Comment:
This functionality needs to be protected against regression with the integration test we discussed.

Reply:
I am not to get you, please elaborate.